client/src/store/modules/tasks.js

```diff
@@ -35,11 +35,11 @@ const getters = {
 
       const allowedTags = ['b', 'i', 'em', 'strong']
       const anchorOptions = {
-        files: false,
-        attributes: [
-          { name: 'target', value: '_blank' },
-          { name: 'rel', value: 'noreferrer' },
-        ]
+        exclude: url => url.toLowerCase().indexOf("file:///") === 0,
```

Comment:
Thanks for your PR!

Is there any reason to exclude the `file:///` urls?

Reply:
The implementation you already have does exclude files, [line38](https://github.com/lessy-community/lessy/pull/396/files#diff-9760e6fe0c6975d0c6533bf543579376L38).

Reply:
That's a good reason, indeed ^_^